diff --git a/user.js b/user.js
index cfc9f59..967213e 100644
--- a/user.js
+++ b/user.js
@@ -640,14 +640,14 @@ user_pref("network.cookie.thirdparty.sessionOnly",		true);
  * SECTION: Caching                                                            *
  ******************************************************************************/
 
-// PREF: Permanently enable private browsing mode
+// PREF: Permanently enable private browsing mode (disabled)
 // https://support.mozilla.org/en-US/kb/Private-Browsing
 // https://wiki.mozilla.org/PrivateBrowsing
-// NOTICE: You can not view or inspect cookies when in private browsing: https://bugzilla.mozilla.org/show_bug.cgi?id=823941
-// NOTICE: When Javascript is enabled, Websites can detect use of Private Browsing mode
-// NOTICE: Private browsing breaks Kerberos authentication
-// NOTICE: Disables "Containers" functionality (see below)
-user_pref("browser.privatebrowsing.autostart",			true);
+// NOTICE-DISABLED: You can not view or inspect cookies when in private browsing: https://bugzilla.mozilla.org/show_bug.cgi?id=823941
+// NOTICE-DISABLED: When Javascript is enabled, Websites can detect use of Private Browsing mode
+// NOTICE-DISABLED: Private browsing breaks Kerberos authentication
+// NOTICE-DISABLED: Disables "Containers" functionality (see below)
+//user_pref("browser.privatebrowsing.autostart",			true);
 
 // PREF: Do not store POST data in saved sessions
 // http://kb.mozillazine.org/Browser.sessionstore.postdata
